department-of-veterans-affairs/vets-website
# MHV-59188 June Staging Review Finding Mismatched URL ls For Conditions Pages

## Summary
I updated VitalListItem.jsx and VitalDetails.jsx to include checks for Pulse, Respiration, and Pulse-oximetry. Once the correct string is identified, the URL conforms to the page title.

## Related issue(s)

[MHV-59188](https://jira.devops.va.gov/browse/MHV-59188) - June Staging Review finding: Mismatched URLs for conditions pages

June Staging Review finding: Mismatched URLs for conditions pages

Details in Github:  

86220-Staging Review finding: Mismatched URLs for conditions pages
https://github.com/department-of-veterans-affairs/va.gov-team/issues/86220

Description
URLs on some conditions history pages does not match h1 on these pages:

Heart rate history (URL slug is /pulse-history)
Breathing rate history (URL slug is /respiration-history)
Blood oxygen level history (URL slug is /pulse-oximetry-history)
Recommended action
Update URL slugs to match the words used in the page H1’s.

User Story: As a Medical Records User, I want to see consistent URL's and content on the conditions page

GIVEN   N/A

WHEN  

THEN  

Feature Flag Y/N ?

DataDog/Analytics Y/N ?

Manual Testing Y/N ?  

Automated Testing Y/N ? 

Accessibility Testing Y/N ? 

UCD Validation Y/N 

Acceptance Criteria:

AC1 URL's no longer mismatched

AC2  Close the GH ticket when resolved

AC3
AC4
AC5

Links to UCD:

Figma Link:

User Flow:

Mobile:

Desktop:

Other Design Notes:

Architectural Notes: Always check for the latest designs in Figma

## Testing done
Conducted UI testing to ensure the expected behavior. I updated existing unit test for new components.

## Screenshots
![Screenshot 2024-07-09 at 2 41 10 PM (2)](https://github.com/department-of-veterans-affairs/vets-website/assets/8038703/04bf25b4-99c4-499b-809a-1365f385318c)
![Screenshot 2024-07-09 at 2 39 48 PM (2)](https://github.com/department-of-veterans-affairs/vets-website/assets/8038703/06913651-1820-4e78-8b9a-a435a1c510e2)
![Screenshot 2024-07-09 at 2 40 00 PM (2)](https://github.com/department-of-veterans-affairs/vets-website/assets/8038703/57afbbe3-3821-47c2-86a1-3b7b9b4915b5)

## What areas of the site does it impact?
The modification affects vitals details section of Medical Records pages.

## Acceptance criteria
Title and URL in vitals detail page matches. 

### Quality Assurance & Testing


### Authentication
